src/runtime/opengl.cpp

```diff
@@ -336,6 +337,17 @@ WEAK char *strndup(const char *s, size_t n) {
     return p;
 }
 
+WEAK char *strstrip(char *str, size_t n) {
+
+  char *pos = str;
+  while (pos != str+n
+         && *pos != '\n'
+         && *pos != ' ')
```

Comment:
Might it be good to add && *pos != '\0' to the condition? Also we use braces on ifs.